src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,148 @@
+package duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Arrays;
+
+public class Parser {
+    public String[] words; // user input split by spaces to get words
+
+    public Parser(String command, TaskList tasks, Storage storage) throws IOException {
+        words = command.split(" "); // split user command
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HHmm");
+        FileWriter fw = new FileWriter("data/duke.txt", true);
+        PrintWriter wf = new PrintWriter(fw);
+
+        // "bye" to end the program
+        switch (words[0]) {
+            case "bye" -> System.out.println("    Bye!! See you again soon!!");
```

Comment:
If I remember correctly, the indentation level should be the same for switch and case statements - you can double check the 2103T standards!